interfaces/PA-App/src/app/models/pa-statuses.enum.ts

```diff
@@ -1,9 +1,3 @@
-export enum PaCredentialStatus {
-  done = 'done',
-  pending = 'pending',
-  noValidation = 'no-validation',
```

Comment:
We could 'reintroduce' these as `PaInclusionStates` like "pending-validation" and "validated" or something?